Octokit.Tests.Integration/Clients/PullRequestsReviewRequestClientTests.cs

```diff
@@ -0,0 +1,164 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Octokit;
+using Octokit.Tests.Integration;
+using Octokit.Tests.Integration.Helpers;
+using Xunit;
+
+public class PullRequestsReviewRequestClientTests : IDisposable
+{
+    private readonly IGitHubClient _github;
+    private readonly IPullRequestReviewRequestsClient _client;
+    private readonly RepositoryContext _context;
+
+    const string branchName = "new-branch";
+    const string collaboratorLogin = "m-zuber-octokit-integration-tests";
+    const string path = "CONTRIBUTING.md";
+
+    public PullRequestsReviewRequestClientTests()
+    {
+        _github = Helper.GetAuthenticatedClient();
+
+        _client = _github.PullRequest.ReviewRequest;
+
+        _context = _github.CreateRepositoryContext("test-repo").Result;
+        _github.Repository.Collaborator.Add(Helper.UserName, _context.RepositoryName, collaboratorLogin);
```

Comment:
When refering to the created `RepositoryContext` I'd prefer we use `_context.RepositoryOwner` rather than `Helper.UserName` - just to keep from making assumptions about the internal workings of repository context